com.github.louis77.tuner.yml

```diff
@@ -0,0 +1,56 @@
+---
+app-id: com.github.louis77.tuner
+runtime: io.elementary.Platform
+runtime-version: '6.1'
+sdk: io.elementary.Sdk
+command: com.github.louis77.tuner
+finish-args:
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--talk-name=org.gtk.vfs"
+- "--talk-name=org.gtk.vfs.*"
+- "--share=network"
+- "--metadata=X-DConf=migrate-path=/com/github/louis77/tuner/"
```

Comment:
```suggestion
```

You don't need this. It's only for apps what used to be available with older runtimes